openfisca-web-api/input-output-data.md

```diff
@@ -94,22 +73,34 @@ Example:
 }
 ```
 
-## variables values
+## axes
 
-A JSON variable value can be:
-* a scalar value: string, integer, boolean. In this case the period is taken in the scenario.
-* a vector: a list of scalar values.
-* an object structured this way:
-  * `<period>` ([JSON period](#periods)): a scalar value or a vector
+A JSON axis is an object structured this way:
+* `count` (integer, >= 1, required): the number of steps to go from min to max
+* `index` (integer, >= 0, default: 0): the index of the person on which to apply the variation of the variable
+* `max` (integer or float, required): the maximum value of the varying variable
+* `min` (integer or float, required): the minimum value of the varying variable
+* `name` (string, one of available [variable names](https://legislation.openfisca.fr/variables), required): the name of the varying variable
+* `period` (a JSON period):
+
+### Parallel axes
 
-JSON variables values can be found in [JSON test cases](#test-cases) or in [JSON input variables](#input-variables).
+TODO
 
-Examples:
+### Perpendicular axes
 
-```json
-20000
-true
-[20, 30, 40]
-{"2014": 5000}
-{"2014-01": [50, 100]}
-```
+TODO
+
+## instants
+
+TODO
+
+## parameters
+
+TODO
+
+## periods
+
+A JSON period can be:
+* a `string` like "2014", "2014-01", "2014-03:2"
+* an `object` like `{start: "2014", unit: "year"}`
```

Comment:
This has been documented properly in #39. Replace this unclear part with a link.